CommunityToolShedMvc/Views/Community/Overview.cshtml

```diff
@@ -0,0 +1,103 @@
+﻿@model CommunityToolShedMvc.ViewModels.CommunityMembersTools
+
+@{
+    ViewBag.Title = Model.Community.Name;
+}
+
+<h2>@Model.Community.Name</h2>
+
+@if (Model.Community.OwnerID != (((CommunityToolShedMvc.Security.CustomPrincipal)User).Person.Id))
+ {
+    <p>The owner of this community is @Model.Community.OwnerName.</p>
+}
+@if (Model.Community.OwnerID == (((CommunityToolShedMvc.Security.CustomPrincipal)User).Person.Id))
+{
+    <p>You are the owner of this community.</p>
+}
+
+<div class="pull-right">
+    @if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Approver"))
+    {
+        <span>You are an Approver</span> 
+        if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Reviewer"))
+        {
+            <span> and Reviewer</span>
+            if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Enforcer"))
+            {
+                <span> and Enforcer</span>
+            }
+        }
+    }
+    else if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Reviewer"))
+    {
+        <span>You are a Reviewer</span>
+        if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Enforcer"))
+        {
+            <span> and Enforcer</span>
+        }
+    }
+    else if (((CommunityToolShedMvc.Security.CustomPrincipal)User).IsInRole(Model.Community.ID, "Enforcer"))
+    {
+        <span>You are an Enforcer</span>
+    }
```

Comment:
You might consider putting these role checks into the view model and adding `IsApprover`, `IsReviewer`, and `IsEnforcer` properties that can be referenced in the view.